tokens_transfer/tokens_transfer.circom

```diff
@@ -34,6 +40,26 @@ template Main(n) {
     var i;
 
     var NONCE_MAX_VALUE = 100;
+
+    // transactions existence check
+    component tx = MultiMiMC7(4,91);
+    tx.in[0] <== pubkey_x;
```

Comment:
We should either 
1. Include the pub x and pub y here 
or 
2. Use the index of the from leaf and then force the person to look that leaf in the tree and get the public key there.

Reply:
let's use pub x and pub y!